browser/about_flags.cc

```diff
@@ -477,6 +482,11 @@ constexpr char kBraveVerticalTabsDescription[] =
      flag_descriptions::kBraveNewsDescription,                              \
```

Comment:
I got a follow up request if you're interested 😄 It would be great to get rid of the original Brave News feature flag. It's disable-able in product now and keeping it as a Chromium feature has caused some recent grief (crashes for NTP when people had news disabled)

Reply:
Oh yeah, that'd be great! I'll keep that in the back of my mind :smile: